readthedocs.yml

```
@@ -6,7 +6,7 @@ build:
     python: "3.10"
   jobs:
     pre_build:
-      - bst -v export --output src/schema.json
+      - bst export src/schema.json
```

Does this need the path to the schema? e.g.
```suggestion
      - bst export src/schema src/schema.json
```

---

I think I figured it out :3